corehq/apps/userreports/reports/view.py

```diff
@@ -359,9 +362,12 @@ class CustomConfigurableReportDispatcher(ReportDispatcher):
 
     @staticmethod
     def _report_class(domain, config_id):
-        class_path = StaticReportConfiguration.report_class_by_domain_and_id(
-            domain, config_id
-        )
+        try:
```

Comment:
yeah, that actually sounds like a good idea since the semantics for ```get_report``` is to return ```None``` if not found.  implemented the suggestion